File: CSharpPartOne/5. ConditionalStatements/06. TheBiggestOfFiveNumbers/TheBiggestOfFiveNumbers.cs
//6. The Biggest of Five Numbers
//Write a program that finds the biggest of five numbers by using only five if statements.

using System;

    class TheBiggestOfFiveNumbers
    {
        static void Main()
        {
            Console.Write("Enter the first number: ");
        double a = double.Parse(Console.ReadLine());
		Console.WriteLine();
		
        Console.Write("Enter the second number: ");
        double b = double.Parse(Console.ReadLine());
		Console.WriteLine();
        Console.Write("Enter the third number: ");
        double c = double.Parse(Console.ReadLine());
		Console.WriteLine();
		Console.Write("Enter the fourth number: ");
        double d = double.Parse(Console.ReadLine());
		Console.WriteLine();
		Console.Write("Enter the fifth number: ");
        double e = double.Parse(Console.ReadLine());
		Console.WriteLine();

        if (a > b && a > c && a > d && a > e)
        {
            Console.WriteLine("The biggest number is: {0}", a);
        }
        else if (b > a && b > c && b > d && b > e)
        {
            Console.WriteLine("The biggest number is: {0}", b);
        }
		else if (c > a && c > b && c > d && c > e)
        {
            Console.WriteLine("The biggest number is: {0}", c);
        }
		else if (d > a && d > c && d > b && d > e)
        {
            Console.WriteLine("The biggest number is: {0}", d);
        }
        else
        {
            Console.WriteLine("The biggest number is: {0}", e);
        }
        }
    }

File: WebDesign/JavaScript/3. Conditional Statements/07.TheBiggestOfFiveNumbers/TheBiggestOfFiveNumbers.js
/* Problem 7. The biggest of five bers
Write a script that finds the greatest of given 5 variables.
Use nested if statements.
Examples:
a	    b	    c	    d	    e	    biggest
5	    2	    2	    4	    1	    5
-2	    -22	    1	    0	    0	    1
-2	    4	    3	    2	    0	    4
0	    -2.5	0	    5	    5	    5
-3	    -0.5	-1.1	-2	    -0.1	-0.1
*/

var a = [5, -2, -2, 0, -3],
    b = [2, -22, 4, -2.5, -0.5],
    c = [2, 1, 3, 0, -1.1],
    d = [4, 0, 2, 5, -2],
    e = [1, 0, 0, 5, -0.1],
    loops = a.length,
    biggest,
    i;

for (i = 0; i < loops; i += 1) {

    console.log('The numbers are: ' + a[i] + ', ' + b[i] + ', ' + c[i] + ', ' +
        d[i] + ' and ' + e[i]);

    biggest = a[i];

    if (b[i] > biggest) {

        biggest = b[i];
    }

    if (c[i] > biggest) {

        biggest = c[i];
    }

    if (d[i] > biggest) {

        biggest = d[i];
    }

    if (e[i] > biggest) {

        biggest = e[i];
    }

    console.log('The biggest number is ' + biggest);

}